doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -9,6 +9,59 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+liblwm2m_carrier 3.3.0
+**********************
+
+Release for modem firmware version 1.3.5 and 2.0.0
+
+Certification status
+====================
+
+For certification status, see `Mobile network operator certifications`_.
+
+Size
+====
+
+See :ref:`lwm2m_lib_size` for an explanation of the library size in different scenarios.
+
++-------------------------+---------------+------------+
+|                         | Flash (Bytes) | RAM (Bytes)|
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (binary)                |               |            |
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (reference application) |               |            |
++-------------------------+---------------+------------+
+
+Changes
+=======
+
+* Added the Binary App Data Container object.
+
+  * The object is enabled for the SoftBank subscriber ID :kconfig:option:`CONFIG_LWM2M_CARRIER_SOFTBANK`, or if the LwM2M carrier library is operating in generic mode :kconfig:option:`CONFIG_LWM2M_CARRIER_GENERIC`.
```

Comment:
```suggestion
* Added the Binary App Data Container object. The object is enabled for the SoftBank subscriber ID (:kconfig:option:`CONFIG_LWM2M_CARRIER_SOFTBANK`), or if the LwM2M carrier library is operating in generic mode (:kconfig:option:`CONFIG_LWM2M_CARRIER_GENERIC`).
```
Since there is just a single entry.